lib/bloc/coins_manager/coins_manager_bloc.dart

```diff
@@ -16,12 +16,12 @@ part 'coins_manager_state.dart';
 class CoinsManagerBloc extends Bloc<CoinsManagerEvent, CoinsManagerState> {
   CoinsManagerBloc({
     required CoinsRepo coinsRepo,
-    required CoinsManagerAction action,
-    required CurrentWalletBloc currentWalletBloc,
+    required KomodoDefiSdk sdk,
   })  : _coinsRepo = coinsRepo,
-        _currentWalletBloc = currentWalletBloc,
-        super(CoinsManagerState.initial(action: action, coins: [])) {
+        _sdk = sdk,
+        super(CoinsManagerState.initial(coins: [])) {
     on<CoinsManagerCoinsUpdate>(_onCoinsUpdate);
+    on<CoinsManagerCoinsListReset>(_onResetCoinsList);
```

Comment:
To be consistent with the conventions and the rest of the handlers, the handler function name should follow `_on{Noun (optional)}{Verb (event)}`

```suggestion
    on<CoinsManagerCoinsListReset>(_onCoinsListReset);
```

Reply:
Renamed along with the formatting changes in [0aaad21](https://github.com/KomodoPlatform/komodo-wallet/pull/2522/commits/0aaad21b396da316dfc3c4b23b735e4dedf0b60f)